pandas/core/generic.py

```diff
@@ -4934,19 +4938,21 @@ def bfill(self, axis=None, inplace=False, limit=None, downcast=None):
             other views on this object (e.g. a column from a DataFrame).
             Returns the caller if this is True.
         limit : int, default None
-            Maximum size gap to forward or backward fill
+            Maximum size gap to forward or backward fill.
         regex : bool or same types as ``to_replace``, default False
             Whether to interpret ``to_replace`` and/or ``value`` as regular
             expressions. If this is ``True`` then ``to_replace`` *must* be a
             string. Alternatively, this could be a regular expression or a
             list, dict, or array of regular expressions in which case
             ``to_replace`` must be ``None``.
-        method : string, optional, {'pad', 'ffill', 'bfill'}
+        method : string, optional, {'pad', 'ffill', 'bfill'}, default is 'pad'
             The method to use when for replacement, when ``to_replace`` is a
             scalar, list or tuple and ``value`` is None.
+        axis : None
+            Deprecated.
 
-        .. versionchanged:: 0.23.0
-           Added to DataFrame
+            .. versionchanged:: 0.23.0
```

Comment:
This isn't what you want to do - make sure you keep the `versionchanged` directive below the `method` argument as that's what was added in v0.23